web/static/util.js

```diff
@@ -714,3 +714,68 @@ function _schemaResolveRefs( a_props, a_refs ){
     }
 }
 */
+
+// Note about the below, I am assuming that the client ID is readily available since it is used in datafed-ws.js
+/**
+ * Basic implementation of get_authorize_url from Globus SDK
+ *  @param {UUID} client_id The UUID of the Globus authentication client
+ *  @param {string} redirect_uri The URI safe application-wide Globus Auth redirect URI.
+ *  @param {Array<string>}requested_scopes The scopes on the token(s) being requested
+ *      In the case of accessing a mapped collection, this should include the mapped collection's UUID
+ *      like so: https://auth.globus.org/scopes/YOUR-UUID-HERE/data_access
+ *  @param {string} state Allows the application to pass information back to itself
+ *  @param {boolean} refresh_tokens Request refresh tokens in addition to access tokens
+ *  @param {object} query_params Additional params
+ *  @returns {string} The URL a user can follow to provide authorization and consent via Globus
+ */
+export function globusGetAuthorizeURL(client_id, redirect_uri, requested_scopes, state, refresh_tokens=false, query_params) {

```

Comment:
This is a quick copy of the functionality here:
https://globus-sdk-python.readthedocs.io/en/stable/services/auth.html#globus_sdk.services.auth.GlobusAuthorizationCodeFlowManager.get_authorize_url